src/WCSimConstructCylinder.cc

```diff
@@ -90,6 +90,18 @@ G4LogicalVolume* WCSimDetectorConstruction::ConstructCylinder()
   if (WCAddGd)
   {water = "Doped Water";}
 
+  // Optionally switch on the checkOverlaps. Once the geometry is debugged, switch off for 
+  // faster running. Checking ALL overlaps is crucial. No overlaps are allowed, otherwise
+  // G4Navigator will be confused and result in wrong photon tracking and resulting yields.
+  // ToDo: get these options from .mac
+  checkOverlaps = true;   
+  checkOverlapsPMT = false; 
```

Comment:
It should when people want to make new geometries to clearly signal any issues, while for current geometries, they should be off or it will generate a lot of output and will seriously slow down running WCSim. Essentially the WCCap has a lot of overlap checks that only need to be done when making a new detector geometry.